src/core/gc/config.d

```diff
@@ -5,7 +5,7 @@
 * License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
 */
 
-module gc.config;
+module core.gc.config;
```

Comment:
Do we want to expose this to the user and docs? Maybe make everything `package` for now?

Reply:
I think a GC implementation should be able to use the configuration even if it does not support all the values in there.

Reply:
> Maybe make everything `package` for now?

That would disallow accessing these modules from package gc AFAICT.

Reply:
> That would disallow accessing these modules from package gc AFAICT.

Fair enough.